internal/cloud/awscloud/awscloud.go

```diff
@@ -332,7 +339,11 @@ func (a *AWS) Register(name, bucket, key string, shareWith []string, rpmArch str
 			Tags: []*ec2.Tag{
 				{
 					Key:   aws.String("Name"),
-					Value: aws.String(name),
+					Value: aws.String(tagName),
+				},
+				{
+					Key:   aws.String("Build-by"),
+					Value: aws.String("osbuild-composer"),
```

Comment:
I'm wondering if the maintenance service could not leverage this new tag instead of the image name for filtering? This would allow us to keep the image name and the value of the `Name` tag consistent. Since you are appending an UUID to the user-provided image name, it will be unique and usable as the single image name.

@croissanne WDYT about using this new tag for the maintenance service?